app/helpers/projects_helper.rb

```diff
@@ -1,2 +1,14 @@
+# frozen_string_literal: true
+
 module ProjectsHelper
+  def total_project_earning(project)
+    project.payments.sum(:amount)
+  end
+
+  def control_links(resource)
```

Comment:
name is very generic, please be a bit specific here, i presume resource is a project, if yes then name it project.

we use such generic words where this resource entry could be of multiple object types.